internal/pipe/project/project.go

```diff
@@ -13,7 +13,11 @@ func (Pipe) String() string {
 // Default set project defaults
 func (Pipe) Default(ctx *context.Context) error {
 	if ctx.Config.ProjectName == "" {
-		ctx.Config.ProjectName = ctx.Config.Release.GitHub.Name
+		if ctx.Config.Release.GitHub.Name != "" {
+			ctx.Config.ProjectName = ctx.Config.Release.GitHub.Name
+		} else if ctx.Config.Release.GitLab.Name != "" {
```

Comment:
can we also add gitea here please?

Reply:
Thanks for the quick response @caarlos0! I've added Gitea, and updated the docs. I had to convert the if-else chain to a switch to keep golangci-lint happy.